src/main/java/com/amazon/janusgraph/batch/BatchCommand.java

```diff
@@ -0,0 +1,54 @@
+package com.amazon.janusgraph.batch;
```

Comment:
Sure, we are adding a scenario test to replicate the issue you encountered and reported in StackOverflow. We are not adding a new tutorial. Repro for issues encountered in the field should be as small as possible for ease of comprehension. Since we are not adding a new tutorial, the new Travel graph does not belong in the main part. If we did put it in the main part, it would just make the distributable JAR artifact larger.